cfme/tests/configure/test_access_control.py

```diff
@@ -50,6 +50,39 @@ def new_role(appliance, name=None):
         vm_restriction='None')
 
 
+@pytest.fixture(scope='module')
+def new_tenant_admin(appliance, request):
+    tenant_collection = appliance.collections.tenants
+    child_tenant = tenant_collection.create(
+        name='child_tenant{}'.format(fauxfactory.gen_alphanumeric()),
+        description='tenant description',
+        parent=tenant_collection.get_root_tenant()
+    )
+    request.addfinalizer(child_tenant.delete_if_exists)
+
+    role = appliance.collections.roles.instantiate(name='EvmRole-tenant_administrator')
```

Comment:
I would break this into smaller fixtures of tenant , group and role and yield them and delete after yield

Reply:
I agree with this ^

Reply:
Done